OutOfSchool/OutOfSchool.Encryption/Handlers/AppHandlers.cs

```diff
@@ -0,0 +1,55 @@
+using Asp.Versioning.Builder;
+using Microsoft.AspNetCore.Mvc;
+using OutOfSchool.Encryption.Models;
+using OutOfSchool.Encryption.Services;
+
+namespace OutOfSchool.Encryption.Handlers;
+
+public static class AppHandlers
+{
+    public static void Map(WebApplication app, ApiVersionSet apiVersionSet)
```

Comment:
Such methods are usually implemented as extension methods (`public static WebApplication Map(this WebApplication app...`), so that they can be chained and called in similar way as standard/other 3rd party implementations.
That's not strict rule, just informaly established convention